### DISCUSSION TITLE
以后能在这出cos吗

### DISCUSSION THREAD

**Discussion by Pray-Fay** [share121/inter-knot#3115] [其他]
![C9E64272519B86728D11001C74AB8AF7](https://github.com/user-attachments/assets/d3adb252-07c7-4aef-9d93-4c595eef080d)

**Comment by x97-1** [comment]
`can can need`

**Comment by seelesee** [comment]
不违反绳网网规就可以的<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >

**Reply by Pray-Fay** [reply]
捉！<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" > 好像在我的第一个帖子下面也是你

**Comment by azb-666** [comment]
包可以的啊

**Comment by langdilaile** [comment]
好看捏

**Comment by Lyfer123** [comment]
最喜欢的一集<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by xxifusi** [comment]
好康，爱康

**Comment by Lyou777** [comment]
请务必上传您的美照<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
